MycroftAI/selene-backend
# Feature/wake word upload

## Description
Implements new database schema, schema changes and a new public API endpoint to support uploading a wake word sample file from a device.

## How to test
Opt-in to data sharing and use wake word to activate device.  Check that reference to file is in the database and file is stored on the server.

## Contributor license agreement signed?
CLA [Y] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
